pb-crm-ui/src/app/contact-book/contact-cache.service.ts

```diff
@@ -44,12 +44,11 @@ export class ContactCacheService {
 
   updateContact(updated: ContactModel): Observable<ContactModel> {
     // Object construction with the spread operator
-    const {id, ...req} = updated;
-    return this.contactService.updateContact(req).pipe(map((contact: ContactModel) => {
-      const filtered = this.remove(id);
+    return this.contactService.updateContact(updated).pipe(map((contact: ContactModel) => {
+      console.log('UPDATED CONTACT: ', updated);
```

Comment:
When it comes to the cache, do you think we should log all updates? Maybe something like

```Typescript
get contacts$() { 
     return this._contacts$().asObservable()
          .pipe(tap(updatedContacts => console.log("New contacts emit", updatedContacts)); 
}

```